### PR TITLE
Break the changes to modular.css

### PR DESCRIPTION
**Pull request by Vidip-Ghosh**
## Related Issue
Broken down the changes of globals.css to modular.css

Closes: #194 

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| Original screenshot | Updated screenshot |

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Comment by Riyazul555** [comment]
Please resolve conflicts and check vercel deployment

File: styles/mentors.module.css [hunk 1]
```diff
@@ -0,0 +1,538 @@
+.mentorForm div {
```
Comment (anmode) [inline styles/mentors.module.css:1]: This file should be further divided.... MentorRegistrationForm , MentorCard. Actually this things is each component should target particular css file.       For a case as we know there is header component so there will be header.css

**Review by Illaf — APPROVED** [review]

File: styles/Properties.module.css [hunk 2]
```diff
@@ -0,0 +1,341 @@
+.loaderBackground {
```
Comment (anmode) [inline styles/Properties.module.css:1]: This file also needed to break

**Review by anmode — CHANGES_REQUESTED** [review]
Nice changes! Well done... But yess Needs more refactoring.. Some class names are not good. Please see needs more modularity and change the class name so it is understandable what does it do

**Comment by Vidip-Ghosh** [comment]
> Nice changes! Well done... But yess Needs more refactoring.. Some class names are not good. Please see needs more modularity and change the class name so it is understandable what does it do

Okay alright. I will make the changes

**Review by Riyazul555 — APPROVED** [review]
Seems fine to me except the upper changes

**Review by muhammadjufry — APPROVED** [review]
All are looking good, only deployment error left

**Comment by anmode** [comment]
There is an error! I checked on local many CSS properties altered. I am worry!!

**Comment by Abhinavcode13** [comment]
deployment shows error here @anmode is it?

**Pull request closed without merging**